test.go

```diff
@@ -1,70 +0,0 @@
-package main
```

Comment:
Could be useful to have an updated test.go file as an example (and something to build and use to ship logs)

Reply:
hmm, how about an `./examples` directory with some sample code?

Reply:
LOG-7109 created for this